make/Makefile.container.mk

```diff
@@ -62,3 +62,35 @@ endif
 
 ## container-push: Pushes all container images to quay
 container-push: container-push-kiali-quay
+
+# Ensure "docker buildx" is available and enabled. For more details, see: https://github.com/docker/buildx/blob/master/README.md
+.ensure-docker-buildx:
+	@if docker buildx version > /dev/null 2>&1 || DOCKER_CLI_EXPERIMENTAL="${DOCKER_CLI_EXPERIMENTAL}" docker buildx version > /dev/null 2>&1 ; then \
+	  echo "'docker buildx' is available and enabled."; \
+	else \
+	  if ! DOCKER_CLI_EXPERIMENTAL="enabled" docker buildx version > /dev/null 2>&1 ; then \
+	    echo "You do not have 'docker buildx'. Note that 'buildx' is only available in Docker 19.03+."; \
+	  else \
+	    echo "You do not have 'docker buildx' enabled by default. Set DOCKER_CLI_EXPERIMENTAL=enabled if you want to use it."; \
+	  fi; \
+	  exit 1; \
+	fi
+#
+
+# Ensure a local builder for multi-arch build. For more details, see: https://github.com/docker/buildx/blob/master/README.md#building-multi-platform-images
+.ensure-buildx-builder:
+	@if [[  ! -f ~/.docker/buildx/instances/kiali-builder ]]; then \
+  		echo "builder 'kiali-builder' not exists. creating 'kiali-builder'"; \
+  		DOCKER_CLI_EXPERIMENTAL="enabled" docker buildx create --name=kiali-builder; \
+  	fi; \
+	if [[ $$(uname -s) == "Linux" ]]; then \
+	  	echo "setup qemu for Linux"; \
+		docker run --privileged --rm tonistiigi/binfmt --install all; \
+	fi; \
+	echo "buildx use 'kiali-builder'"; \
+	DOCKER_CLI_EXPERIMENTAL="enabled" docker buildx use kiali-builder
```

Comment:
Added .ensure-buildx-builder too.

For Mac user. The default builder couldn't using `--push`. need to create another builder. but multi arch supports provided by Docker for mac.

For Linux user.  need `qemu` to support multi-arch.